ros/angel_utils/scripts/bag_extractor.py

```diff
@@ -130,7 +130,7 @@ def __init__(self):
             .string_value
         )
         self.pv_image_frame_id = (
-            self.declare_parameter("pv_image_frame_id", "PVFramesBGR")
+            self.declare_parameter("pv_image_frame_id", "/kitware/pv_image_detections_2d")#"PVFramesBGR")
```

Comment:
I would have expected this to just need to be `"pv_image_detections_2d"`, and the namespace under which the bag play happens does appropriate namespace prepending to messages emitted?